src/System.Windows.Extensions/tests/System/Media/SoundPlayerTests.cs

```diff
@@ -143,6 +143,26 @@ public async Task LoadAsync_SourceLocationFromNetwork_Success(string sourceLocat
             player.Play();
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsSoundPlaySupported))] 
+        [OuterLoop]
+        public void Play_InvalidFile_ShortTimeout_ThrowsWebException()
+        {
+            var player = new SoundPlayer();
+
+            using (Socket listener = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
+            {
+                listener.Bind(new IPEndPoint(IPAddress.Loopback, 0));
+                listener.Listen(1);
+                var ep = (IPEndPoint)listener.LocalEndPoint;
+
+                var tcs = new TaskCompletionSource<AsyncCompletedEventArgs>();
+                player.LoadCompleted += (s, e) => tcs.TrySetResult(e);
```

Comment:
You can delete the above two lines.  You can also move the `var player = new SoundPlayer()` to here.